packages/toolpad-core/src/shared/Link.tsx

```diff
@@ -5,12 +5,13 @@ import { RouterContext } from './context';
  * @ignore - internal component.
  */
 
-export interface LinkProps extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
+export interface DefaultLinkProps extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
```

Comment:
Can't this just be the `LinkProps` that we use both for `DefaultLink` and `Link`?
```suggestion
export interface LinkProps extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
```